src/validation.cpp

```diff
@@ -4720,6 +4708,46 @@ const AssumeutxoData* ExpectedAssumeutxo(
     return nullptr;
 }
 
+static bool DeleteCoinsDBFromDisk(const fs::path db_path, bool is_snapshot)
```

Comment:
note for reviewers: for now `is_snapshot` is always true as there is a single usage. This is not the case anymore with the remaining commits of the #24232 patchset.